.circleci/config.yml

```diff
@@ -245,41 +274,118 @@ jobs:
             ./src/test/scripts/check_enterprise_merge.sh
 
 
+orbs:
+  codecov: codecov/codecov@1.0.5
+  azure-cli: circleci/azure-cli@1.0.0
 
 workflows:
   version: 2
+
+  performance_tests:
+    jobs:
+      - ch_benchmark:
+          filters:
+            branches:
+              only:
+                - /ch_benchmark\/.*/ # match with ch_benchmark/ prefix
+      - tpcc_benchmark:
+          filters:
+            branches:
+              only:
+                - /tpcc_benchmark\/.*/ # match with tpcc_benchmark/ prefix
+
   build_and_test:
     jobs:
+      - build:
+          filters:
+            branches:
+              ignore:
+                - /(tpcc|ch)_benchmark\/.*/ # ignore tpcc_benchmark/ and ch_benchmark/
+      - check-style:
+          filters:
+            branches:
+              ignore:
+                - /(tpcc|ch)_benchmark\/.*/ # ignore tpcc_benchmark/ and ch_benchmark/
+      - check-sql-snapshots:
+          filters:
+            branches:
+              ignore:
+                - /(tpcc|ch)_benchmark\/.*/ # ignore tpcc_benchmark/ and ch_benchmark/
       - check-merge-to-enterprise
-      - build
-      - check-style
-      - check-sql-snapshots
 
       - test-11_check-multi:
           requires: [build]
+          filters:
```

Comment:
Well, 

- Typically we expect that the benchmark runs will be very rare
- Most of the time we also want to see whether the branch passes all other regression tests as well, like we're doing some changes and we want to see if it brakes anything or not as well.
- The complexity of this config.yml increases **a lot** with these ignores

I'd rather run remove these changes and let the CI run the benchmarks along with the other checks. Plus, undo any other changes on this file that is not relevant to this change. For example, why do we move code-cov line to somewhere else?

Reply:
>Most of the time we also want to see whether the branch passes all other regression tests as well, like we're doing some changes and we want to see if it brakes anything or not as well.

I think the workflow will be that someone will be doing some changes/improvements etc. in their branch,say `X`. Then when they want to see the performance changes, they will create a new branch, say `tpcc_benchmark/X`. So since `X` was already pushed it should be passing the `build` job etc. If it is not passing the `build` job, the benchmark will fail. Actually it might make sense to make the `build` job required for these two benchmark jobs. Because we should not trigger the jobs if build is failing.(otherwise in the runtime it will already fail).

Because if someone starts developing with `tpcc_benchmark/X` branch then every time they push to the branch, a new hammerdb job will be triggered and if the previous one didn't finish it will fail.

The reason code-cov line changed was that I added another orbs `az-cli`. But of course there is not reason for it to change, so will put it back to the top.

I agree that config.yml's complexity increases,  the `workflow` part is not something we play a lot with.

So I am kind of neutral on this

Reply:
> So I am kind of neutral on this

Given the cost of re-running the CI is not that much, and you don't have strong opinions on this, I'd rather to keep the config simpler and remove these filter.

Reply:
Sure, I am okay with both. @marcocitus  do you have any comments since you suggested disabling the other tests?

Reply:
Removed the filters and made the `build` job required for the benchmark jobs.